tools/slintpad/README.md

```diff
@@ -55,3 +54,5 @@ The `preview.html` page contains only the preview and the code must be given via
 
     -   https://slint.dev/editor/?snippet=_+%3A%3D+Text+%7B+text%3A+%22Hello+Slint%22%3B+%7D?style=fluent-dark
     -   https://slint.dev/editor/preview.html?snippet=_+%3A%3D+Text+%7B+text%3A+%22Hello+Slint%22%3B+%7D?style=material
+
+-   `?s=` Like `?snippet=` but compressed with gzip and base64 encoded.
```

Comment:
I use `s` because it is short for `snippet`, but I'm actually open for other name.
maybe `c` for compressed, or `csnippet`.  or `z` for "zip". or maybe `zip` or `gzip`?